asset-properties-widget/component/asset-properties-config/asset-property-selector/asset-property-selector.component.ts

```diff
@@ -142,10 +143,10 @@ export class AssetPropertiesSelectorComponent {
     }
 
     updateOptions(){
-      if(this.asset && this.properties.map(function(item) { return item.active}).indexOf(true) > -1){
-        this.contextDashboardService.formDisabled = false;
+      if(this.properties.map(function(item) { return item.active}).indexOf(true) > -1){
+        this.isPropertySelected = true;
       }else {
-        this.contextDashboardService.formDisabled = true;
+        this.isPropertySelected = false;
```

Comment:
Can simplify this without using lodash as
```
updateOptions() {
 this.isPropertySelected = _.some(this.properties, 'active');
}

```

Reply:
incorporated.